eve_sqlalchemy/tests/integration/collection_class_set.py

```diff
@@ -84,3 +85,9 @@ def test_patch_parent(self):
         self.assert200(status)
         response, _ = self.get('parents', item=2)
         self.assertEqual(response['children'], [3, 4])
+
+    def test_get_can_return_none_value(self):
+        response, status = self.get('parents/1')
+        self.assert200(status)
+        self.assertIn('none_field', response)
+        self.assertIsNone(response['none_field'])
```

Comment:
Consider moving this test to its own test case as it's not really related to collection classes. How about `tests/integration/get_none_values.py`?